src/pages/index.tsx

```diff
@@ -69,7 +75,36 @@ export default function Page() {
           on AWS. Get started by selecting your preferred framework.
         </Text>
       </Flex>
-
+      <LinkCardCollection>
+        <LinkCard
+          isExternal={true}
+          href={''}
+          icon={() => <IconGithub fontSize="2rem" />}
+        >
+          {'JavaScript Libraries on Github'}
+        </LinkCard>
+        <LinkCard
+          isExternal={true}
+          href={''}
+          icon={() => <IconDiscord fontSize="2rem" />}
+        >
+          {'Amplify Discord'}
+        </LinkCard>
+        <LinkCard
+          isExternal={true}
+          href={''}
+          icon={() => <IconAmplify fontSize="2rem" />}
+        >
+          {"What's next for Amplify"}
```

Comment:
brackets and quotes are unnecessary
```suggestion
What's next for Amplify
```

Reply:
Same feedback as above